lambdas/evaluate-gpg45-scores/src/main/java/uk/gov/di/ipv/core/evaluategpg45scores/domain/CredentialEvidenceItem.java

```diff
@@ -37,6 +39,9 @@ public Gpg45Scores.Evidence getEvidenceScore() {
     }
 
     public boolean hasContraIndicators() {
+        if (isIdentityFraud()) {
+            return ci != null && !ci.isEmpty() && !(ci.size() == 1 && ci.get(0).equals(A01));
```

Comment:
This is fine for now - but `.get(o)` raises a my eyebrows a little. I will have a think about how we can make this "safer".